### PR TITLE
148 integrate vanity and ar5iv

### PR DESCRIPTION
**Pull request by julien-blanchon**

### CODE REVIEW

File: src/shared/js/utils/functions.js [hunk 1]
```diff
@@ -441,6 +441,36 @@ const tablerSvg = (pathName, id, classNames) => {
...
+                <path d="M 35.767663 59.686141 L 35.633152 48.754076 C 35.461957 47.519022 34.373641 46.601902 33.126359 46.638587 C 31.842391 46.589674 30.717391 47.482337 30.497283 48.754076 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 25.251359 55.235054 L 25.116848 39.142663 C 24.945652 37.907609 23.857337 37.002717 22.610054 37.039402 C 21.326087 36.978261 20.201087 37.883152 19.980978 39.154891 L 20.005435 44.755435 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 38.995924 36.5625 L 45.415761 26.902174 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 36.085598 43.532609 C 36.085598 43.532609 37.956522 42.138587 38.519022 41.612772 C 39.558424 40.78125 40.120924 39.497283 39.998641 38.164402 C 39.839674 37.173913 38.922554 36.501359 37.932065 36.648098 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 43.153533 25.410326 L 52.813859 11.017663 L 58.035326 13.769022 L 48.203804 28.748641 L 45.415761 26.902174 Z M 43.153533 25.410326 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+            </svg>`;
+
+        case "ar5iv":
```
Comment (julien-blanchon) [inline src/shared/js/utils/functions.js:460]: It's pretty ugly, maybe remove it and just use "message"

File: src/shared/js/utils/functions.js [hunk 2]
```diff
@@ -441,6 +441,36 @@ const tablerSvg = (pathName, id, classNames) => {
                 <path d="M14 15v2a1 1 0 0 1 -1 1h-7l-3 3v-10a1 1 0 0 1 1 -1h2" />
             </svg>`;
 
+        case "vanity":
```
Comment (julien-blanchon) [inline src/shared/js/utils/functions.js:444]: Not perfect

File: src/popup/js/templates.js [hunk 3]
```diff
@@ -294,6 +294,32 @@ const getPopupPaperIconsHTML = (paper, currentUrl, is) => {
         </div>`;
     }
 
+    let vanity = "";
```
Comment (julien-blanchon) [inline src/popup/js/templates.js:297]: Why not add SciRate, Vanity and Ar5iv to the global view (not popup): line 100 (getMemoryItemHTML)

File: src/shared/js/utils/functions.js (hunk 2 above)
Comment (vict0rsch) [reply src/shared/js/utils/functions.js:444]: No but close enough! I would keep that version :)

File: src/shared/js/utils/functions.js (hunk 1 above)
Comment (vict0rsch) [reply src/shared/js/utils/functions.js:460]: yeah that one's too much.

I think `square-rounded-number-5` from https://tabler-icons.io/ would be good enough

File: src/popup/js/templates.js (hunk 3 above)
Comment (julien-blanchon) [reply src/popup/js/templates.js:297]: Not especially, let's keep it simple for now and maybe figure out a better way to integrate if this is wanted by others than just me ;)

**Comment by julien-blanchon** [comment]
> I'll probably have to work on parsing those sources too (e.g. https://ar5iv.labs.arxiv.org/html/2005.01643). Except if you want to, for the sake of the exercise or if I'm too slow 😛

By parsing you mean add this as a paper source ?

**Comment by vict0rsch** [comment]
Yes exactly. It's messier

**Comment by vict0rsch** [comment]
I've resolved the conflict in `templates.js` due to #161. The minified files are still the ones from the branch. If you pull, build locally, push back it should be fine.

**Comment by vict0rsch** [comment]
@julien-blanchon I think we're good now, I added the svgs to the memory items (in the list view) and added the parsing from vanity ar5iv and scirate to the `arxiv` source. 

FYI I'll merge at the end of the week because I want to release quickly :)

**Comment by vict0rsch** [comment]
I totally understand, you're 100% entitled to vacation haha it's just for me to move on. 

w.r.t. sources, yes, in `config.js` I treat them as `arxiv` URL patterns:

```
    arxiv: [
        "arxiv.org/abs/",
        "arxiv.org/pdf/",
        "scirate.com/arxiv/",
        "ar5iv.labs.arxiv.org/html/",
        "arxiv-vanity.com/papers/",
    ],
```

**Pull request merged**